site/documentation/content/user-guide/mqtt-adapter.md

```diff
@@ -621,7 +621,17 @@ Example payload:
 When a device publishes a telemetry, event or command response message and there is an error processing the message, the handling of the error depends on whether there is a [error topic subscription]({{< relref "#error-reporting-via-error-topic" >}}) for the device and whether a *on-error* property bag parameter was set on the topic used for sending the message.
 
 If no error subscription is in place and no *on-error* parameter was set, the default error handling behaviour is to close the MQTT connection to the device.
-If the device has a subscription on the error topic (on the same MQTT connection the device uses for sending messages), the default behaviour is to keep the MQTT connection open. 
+If the device has a subscription on the error topic (on the same MQTT connection the device uses for sending messages), the default behaviour is to keep the 
+MQTT connection open unless a terminal error happens. The errors that are classified as terminal are listed below.
+
+* The adapter is disabled for the tenant that the client belongs to.
+* The device is disabled or not registered (provided that the device is not connected via a gateway).
+* The gateway is disabled or not registered.
```

Comment:
I think replacing the above 2 entries with
`The authenticated device or gateway is disabled or not registered.`
would be easier to understand.
(I think in case an *unauthenticated* device has sent a message, the connection shouldn't be disconnected anyway - see my other comment.)

Reply:
The unauthenticated devices are not disconnected unless they are disabled or not registered, see the below comment.